spec/model_spec.cr

```diff
@@ -9,6 +9,29 @@ private class QueryMe < LuckyRecord::Model
   end
 end
 
+private class MissingColumn < LuckyRecord::Model
+  table users do
+    field missing : String
+  end
+end
+
+private class OptionalFieldOnRequiredColumn < LuckyRecord::Model
+  table users do
+    field name : String?
+  end
+end
+
+private class RequiredFieldOnOptionalColumn < LuckyRecord::Model
+  table users do
+    field nickname : String
+  end
+end
+
+private class MissingTable < LuckyRecord::Model
+  table uusers do
```

Comment:
`uusers` looks like an undeliberate typo :) maybe name it `intentionally_missing_table` haha :)

Reply:
lol good point, I'll change that

Reply:
Ah I just realized, the reason I made it similar to users is because it checks for similarity with `Levenshtein` to suggest a correction